srcpkgs/python-markdown-math/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'python-markdown-math'
+pkgname=python-markdown-math
```

Comment:
should probably be namend python3-markdown-math

Reply:
Renamed.